cluster-scope/base/podsecuritypolicy/metallb/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
No resources in this file.

Each folder in `cluster-scope/base/RESOURCE_PLURAL/` should be named the same as the resource `.metadata.name` and implement exactly one resource. See:
https://github.com/operate-first/apps/blob/master/cluster-scope/README.md